### PR TITLE
Fixed a bug allowing the user to pick an unsupported glyph

### PR DESCRIPTION
**Pull request by Aldrian**
Fixed a bug allowing the user to pick an unsupported glyph, added a feedback when the user tries to input a non-available glyph
image

### CODE REVIEW

File: app/styles/components/canvas-glyph-input.scss [hunk 1]
```diff
@@ -42,6 +42,10 @@
 				color: $theme-color;
 				background-color: $theme-background;
 				border-color: $theme-background;
+				transition-duration: .2s;
```
Comment (FranzPoize) [inline app/styles/components/canvas-glyph-input.scss]: Is it on purpose that it takes as much time for the transition to end that for the timeout to trigger. I think a longer would be good.

Comment (FranzPoize) [inline app/styles/components/canvas-glyph-input.scss]: longer timeout

Comment (Aldrian) [inline app/styles/components/canvas-glyph-input.scss]: Yup, that's expected. This way, the error color will never stay, it'll always be transitioning.
If I double the timeout value and an user keeps spamming an incorrect input, the red highlighting will be kinda shaky.

Comment (FranzPoize) [inline app/styles/components/canvas-glyph-input.scss]: Ok i'll test it out

**Pull request merged**